application/sitebuilder/build_service.py

```diff
@@ -67,6 +73,7 @@ def s3_deployer(app, build_dir, deletions=[]):
         to_delete = list(bucket.objects.filter(Prefix=prefix))
 
         for d in to_delete:
+            print("Deleting: " + d.key)
```

Comment:
Noisy is fine, but I'm pretty sure we never really have anything to delete here :man_shrugging:

Reply:
@TheDoubleK it should delete pages, for example, when we remove the 'static' pages in the "Ethnicity in the UK" section?

Reply:
Only unpublished measures. So it won't do the ethnicity in UK pages.